### ISSUE TITLE
possbile translation for spanish

### ISSUE THREAD

**Issue by johnfelipe** [regardscitoyens/the-law-factory#124]
Transifex is possible?

**Comment by RouxRC** [comment]
Hi @JohnFelipe,

@pserwylo already started some work in a fork here to add i18n for english https://github.com/regardscitoyens/the-law-factory/compare/master...pserwylo:feature/i18n

I believe lots of pieces are still missing, but that seems like a good way to go with these json locale translations files. Including spanish as well would certainly be very welcome.

Although I'm not sure transifex is really necessary yet: not having that many contributors, I guess github contributions are reasonable for now?

**Comment by johnfelipe** [comment]
Hi bro, me again, pls tell me if can i translate all site to spanish and contribute, may be a .po file or strings
Stay alert

**Comment by RouxRC** [comment]
Hi @johnfelipe , unfortunately we haven't advanced at all on this part since. The best effort so far for internationalization is the above quoted fork, which would probably require some fixes following some code changes since.

**Comment by pserwylo** [comment]
Yeah, I really wish I didn't leave that branch to rot. I'm still very interested in this idea of a UI for lays and amendments, but haven't had the time/motivation to continue that fork. If somebody wanted to work on a translation for this, and was able to break it down into a few tasks (e.g. 1st add the dependencies for i18n, 2nd i18n this particular page (e.g. a simple about page or such)), then I would like to try and find some time to help work on some of those tasks and get this internationalized.

**Comment by johnfelipe** [comment]
some advance?
